### DISCUSSION TITLE
Why gitignore?

### DISCUSSION THREAD

**Discussion by jochemvankessel** [vercel/turborepo#6676] [Help] [answered]
Just curious: why do we need to copy the gitignore? What is the purpose of this?

https://github.com/vercel/turbo/blob/a8859c787300e1f4451e6e836e31cc4e89c63613/examples/with-docker/apps/web/Dockerfile#L23

**Comment by anthonyshew** [comment, marked as answer]
Turborepo relies on git to learn about the file change state of your repository.
